client/src/components/end-screen/end-screen.vue

```diff
@@ -3,14 +3,27 @@ import { computed } from 'vue';
 import { default as Button } from '../generic-button/generic-button.vue';
 import { useChannelStore } from '../../stores/channel';
 import BigNumber from 'bignumber.js';
-import { useTransactionsStore } from '../../stores/transactions';
 import { GameChannel } from '../../utils/game-channel/game-channel';
+import { TransactionLog } from '../transaction/transaction.vue';
+import { useTransactionsStore } from '../../stores/transactions';
 
 const channel = useChannelStore().channel as GameChannel;
 
-const transactions = useTransactionsStore()
-  .userTransactions.flat()
-  .filter((tx) => tx.onChain === false);
+let offChainTxNo = 0;
+if (channel.didReconnect) {
```

Comment:
```suggestion
let offChainTxNo = 0;
const didReconnect = !!localStorage.getItem('gameState');
if (didReconnect) {
```
No need for it to be ref as we will only check it on first render

Reply:
What ref?